src/masternode/masternode-meta.h

```diff
@@ -35,7 +35,6 @@ class CMasternodeMetaInfo
 
 public:
     CMasternodeMetaInfo() {}
-    CMasternodeMetaInfo(const uint256& _proTxHash) : proTxHash(_proTxHash) {}
```

Comment:
you remove it here and then re-add it in the next commit?

Reply:
Must have realized it was actually used and maybe did a soft reset instead of a hard reset, imo it's mostly fine since this will all be squashed into one merge

Reply:
True, ignore my comment :+1: